llm_on_ray/finetune/finetune.py

```diff
@@ -155,6 +155,10 @@ def train_func(config: Dict[str, Any]):
 
     gradient_accumulation_steps = config["Training"].get("gradient_accumulation_steps", 1)
     base_model = config["General"]["base_model"]
+    if config["General"].get("tokenizer_name") is not None:
+        tokenizer_name = config["General"].get("tokenizer_name")
+    else:
+        tokenizer_name = base_model
```

Comment:
```suggestion
    tokenizer_name = config["General"].get("tokenizer_name", base_model)
```

Reply:
If the 'tokenizer_name' is not specified in the config YAML, the expression config["General"].get("tokenizer_name") will return None instead of the expected default value 'base_model'. I plan to submit a pull request to address this issue.